src/libraries/System.Linq/tests/AnyTests.cs

```diff
@@ -59,7 +59,7 @@ public static IEnumerable<object[]> TestData()
 
         [Theory]
         [MemberData(nameof(TestData))]
-        public void Any(IEnumerable<int> source, bool expected)
+        public void AnyTest(IEnumerable<int> source, bool expected)
```

Comment:
This should stay `Any`, and the test below should be changed to be `Any_Predicate`, or something like that.